app/Http/Controllers/CompetitionsController.php

```diff
@@ -51,12 +44,9 @@ public function create()
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(CompetitionRequest $request)
```

Comment:
You need to import this class into this file with a `use` statement. Otherwise it'll be looking for it in the current namespace (e.g. `Gladiator\Http\Controllers\CompetitionsRequest`).

`use Gladiator\Http\Requests\CompetitionRequest;`